mock/py/mockbuild/backend.py

```diff
@@ -666,14 +667,27 @@ def install_srpm(self, srpm_path):
         if return_code:
             raise PkgError("Source RPM is not installable:\n{0}".format(output))
 
+
+    @property
+    def _rpmbuild_noclean_option(self):
```

Comment:
Can you add FIXME comment here that this method and its use can be removed after some date?